ui/packages/consul-partitions/app/templates/dc/partitions/index.hbs

```diff
@@ -48,7 +48,15 @@ as |route|>
         </h1>
       </BlockSlot>
       <BlockSlot @name="actions">
-          <a data-test-create href="{{href-to 'dc.partitions.create'}}" class="type-create">Create</a>
+{{#if (can 'create partitions')}}
+        <a 
```

Comment:
For context since I'm new, why is consul not using `LinkTo`?

Reply:
It began as "because it's heavier than href-to" apparently technically but also "code-visually" (bad word but not sure how to explain that one) i.e. I preferred to just use an `<a>`. I also think maybe we've dodged some upgrade bullets due to that but not entirely sure.

More importantly, as we've had to add some of our custom routing things its be super helpful to have a better/customizable point to tweak things we needed to do (mainly around the optional routing/URL segments we have).

Longer term, we completely own `href-to` in our codebase now I believe and at some point soon I'd like to add relative routeName ability to it so we have 100% isolated routes that can be dropped in anywhere and "Just Work(tm)" (supposedly 😆 🤞 )